### PR TITLE
Docs for Cartopy 0.17

### PR DESCRIPTION
**Pull request by dopplershift**

### CODE REVIEW

**Comment by dopplershift** [comment]
Ping @kaedonkers

File: cartopy/docs/v0.17/_static/version_switch.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+// Copyright 2013 PSF. Licensed under the PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2
+// File originates from the cpython source found in Doc/tools/sphinxext/static/version_switch.js
+
+(function() {
+  'use strict';
+
+  var all_versions = {
+    'latest': '0.15',
```
Comment (QuLogic) [inline cartopy/docs/v0.17/_static/version_switch.js]: This needs to be updated as well.

Comment (dopplershift) [reply cartopy/docs/v0.17/_static/version_switch.js]: Fixed, thanks.

**Comment by QuLogic** [comment]
Well, I don't see any reason not to merge this as I see people using Cartopy 0.17 out in the wild already.

**Pull request merged**